css/menu.css

```diff
@@ -1,17 +1,21 @@
 /* MENU */
+[data-fl-widget-instance][data-widget-package="com.fliplet.menu.default"] {
```

Comment:
I don't think the `com.fliplet.menu.default` part is necessary because it's not like you'll ever have 2 menus. If you want to target menus only instead of other screen components, perhaps add `data-type="menu"` if that's relevant.